coldfront/plugins/qumulo/views/update_allocation_view.py

```diff
@@ -124,6 +185,11 @@ def form_valid(
         # needed for redirect logic to work
         self.success_id = str(allocation.id)
 
+    def form_valid(self, form: UpdateAllocationForm):
```

Comment:
Should be moved higher up on the page for readability.  When I'm looking over a view, I'm looking for this method because I know django is calling it during the submission flow.